Tests/TestBase.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Infrastructure.Pages;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using Core;
+
+namespace Tests
+{
+    [TestClass]
```

Comment:
This annotation is not necessary in a class where you have test initi/cleanup as far as I remember